src/test/integration/java/pl/palonek/interview/InterviewApplicationTests.java

```diff
@@ -0,0 +1,124 @@
+package pl.palonek.interview;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpEntity;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringRunner;
+import pl.palonek.interview.course.model.CourseSessionDTO;
+import pl.palonek.interview.course.model.CourseSummary;
+import pl.palonek.interview.course.model.SessionStatus;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+        classes = InterviewApplication.class)
+@AutoConfigureMockMvc
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+public class InterviewApplicationTests {
+
```

Comment:
+1 I like how the test was implemented.
-1 There is only happy-path testing and it would be useful to implement unit test for services.